src/playwright/github/jobs/setup-node.ts

```diff
@@ -0,0 +1,106 @@
+import type {JobStep} from 'projen/lib/github/workflows-model'
```

Comment:
The same as `node-job-options.ts` - remove the duplicate.

Reply:
Fixed